### ISSUE TITLE
Prevent sending values if update too frequent

### ISSUE THREAD

**Issue by WinndyBoy** [mysensors/NodeManager#313]
I'm updating the SensorDimmer->setPercentage() when a button is held, and currently a message is being sent to the network on every change.

I suggest enhancing the Child::sendValue some how so that if values change quicker than say 5 seconds (configurable) then these messages arn't sent until things have settled. Perhaps the SensorDimmer is a special case and sending messages can be prevented somewhere else, but though I'd mention it for feedback.

**Comment by user2684** [comment]
I think it makes sense but I'd not touch Child; probably I'd add a sort of debounce at the specific sensor which may suffer from this. Thanks!

**Comment by WinndyBoy** [comment]
Thanks for the confirmation, thats what I've done with the SensorDimmer and probably I'll take my changes out and put them into an extended class. Can share later what I did. By sort of debounce do you mean a timer which waits until no changes are made for say 5 seconds and only then writes the value to the ChildInt or other ? This is what I did... or is there another simpler way.

**Comment by user2684** [comment]
Yes, something like setInterruptDebounce() from #322 which is basically ignoring further interrupts for a given timeframe
